src/parser/paladin/protection/modules/spells/Judgment.js

```diff
@@ -24,13 +35,51 @@ class Judgment extends Analyzer {
       return;
     }
 
+    this._casts += 1;
+
+    const isCrit = event.hitType === HIT_TYPES.CRIT || event.hitType === HIT_TYPES.BLOCKED_CRIT;
+    this._crits += isCrit ? 1 : 0;
+
     if (this.spellUsable.isOnCooldown(SPELLS.SHIELD_OF_THE_RIGHTEOUS.id)) {
       // Nope, I did not verify if blocked crits count as crits for this trait, I just assumed it. Please do test if you can and report back or fix this comment.
       // Confirmed: Blocked crits count as crits.
-      const isCrit = event.hitType === HIT_TYPES.CRIT || event.hitType === HIT_TYPES.BLOCKED_CRIT;
       const reduction = isCrit ? REDUCTION_TIME_CRIT : REDUCTION_TIME_REGULAR;
-      this.spellUsable.reduceCooldown(SPELLS.SHIELD_OF_THE_RIGHTEOUS.id, reduction);
+      const actualReduction = this.spellUsable.reduceCooldown(SPELLS.SHIELD_OF_THE_RIGHTEOUS.id, reduction);
+      this._totalCdr += actualReduction;
+      this._wastedCdr += reduction - actualReduction;
+    }
+  }
+
+  get cdrPercentage() {
+    return this._totalCdr / (this.owner.fightDuration + this._totalCdr);
+  }
+
+  // cdr percentage without extra casts from Crusader's Judgment
+  get baseCdr() {
+    if(!this.selectedCombatant.hasTalent(SPELLS.CRUSADERS_JUDGMENT_TALENT.id)) {
+      return this._totalCdr;
     }
+    
+    return (1 - this.gc._totalResets / this._casts) * this._totalCdr;
+  }
+
+  get baseCdrPercentage() {
+    return this.baseCdr / (this.owner.fightDuration + this.baseCdr);
+  }
+
+  statistic() {
+    const cjTooltip = this.selectedCombatant.hasTalent(SPELLS.CRUSADERS_JUDGMENT_TALENT.id) ?
+      `<br/>Without the Crusader's Judgment talent, this would have been roughly <b>${formatPercentage(this.baseCdrPercentage)}%</b> (or ${formatNumber(this.baseCdr / 1000)}s).` : '';
+    return (
+      <StatisticBox
```

Comment:
Is this meant to say x% less?

Reply:
No, it says what it says it says ;)

(see the pic: that line says 30.11% CDR w/o CJ, compared to the 33.96% with it)

Reply:
I think it would be clearer what is meant if it said:
```Without the Crusader's Judgment talent, your effective SotR CDR would have been roughly <b>${formatPercentage(this.baseCdrPercentage)}%</b> (or ${formatNumber(this.baseCdr / 1000)}s).```
But that may just be me? - also your attempted changelog conflict fix didn't work 😢

Reply:
that makes sense :+1:

also rip, guess we go back to the old way of dealing with changelog conflicts :(